Rohit/PostgresAssignment/src/app/server.ts

```diff
@@ -0,0 +1,22 @@
+import { router } from "../router/router";
```

Comment:
why 2 server.ts ? remove duplicate/ unwanted files and logs.

Reply:
resolved